src/__tests__/patients/related-persons/RelatedPersons.test.tsx

```diff
@@ -78,34 +86,36 @@ describe('Related Persons Tab', () => {
       expect(newRelatedPersonModal.prop('show')).toBeTruthy()
     })
 
-    it('should call update patient with the data from the modal', () => {
+    it('should call update patient with the data from the modal', async () => {
       jest.spyOn(patientSlice, 'updatePatient')
       jest.spyOn(PatientRepository, 'saveOrUpdate')
       const expectedRelatedPerson = { patientId: '123', type: 'type' }
       const expectedPatient = {
         ...patient,
         relatedPersons: [expectedRelatedPerson],
       }
-
-      act(() => {
-        const newRelatedPersonButton = wrapper.find(Button)
-        const onClick = newRelatedPersonButton.prop('onClick') as any
-        onClick()
-      })
-
-      wrapper.update()
-
-      act(() => {
-        const newRelatedPersonModal = wrapper.find(NewRelatedPersonModal)
-
-        const onSave = newRelatedPersonModal.prop('onSave') as any
-        onSave(expectedRelatedPerson)
+      await act(async () => {
```

Comment:
What is the reason for this test getting updated?

Reply:
I was getting errors and there was an async operation that used the wrappers which I was creating with the setup function so I made setup async and adapted the rest of the file. The test is indeed missing the new functionality, I refactored all the tests for the changes but forgot to add the new one. If it would be more performant and reusable we make the beforeeach but I think we with the setup wrapper we can call it once on the describe scope instead of each test and in cases where we need a new patient user or route we call the setup function in the individual test. Give me a heads up and I'll finish refactoring this as necessary